release/models/aft/openconfig-aft-common.yang

```diff
@@ -342,6 +348,18 @@ submodule openconfig-aft-common {
         "The protocol from which the AFT entry was learned.";
     }
 
+    leaf preference {
```

Comment:
Today, the AFT model exposes only the selected routes -- and must because there's no ability to have >1 entry with the same key. I'd just like the clarify that the intention is to return just the admin distance/preference of the AFT entry that is active, we're not trying to change the original semantics here.

Reply:
That's correct - one part is the returned state but the other intention was since this model is imported into the stub model that generates the proto IDL for gRIBI programming that it could be utilized to "set" the preference and/or metric without modifying the gRIBI APIs directly.

If however as you mention in the other comment that gRIBI is to have a single preference/metric combo that is protocol wide and not per entry, then this would solely then be used for state representation however the driver behind this was for gRIBI.

If gRIBI is going to have a single preference/distance then we will want to discuss how this is driven (likely `/system/` schema config) and preferred order since various implementations can differ in preference/admin-distance treatment (e.g. after static/connected but prior to protocols, etc..)

Reply:
@robshakir - any additional thoughts or concerns on this?

Reply:
Per discussion - removed AFT related parameters.  Will bring in gRIBI preference related via separate PR.  Updated in latest commit